foxy/distribution.yaml

```
@@ -611,6 +611,11 @@ repositories:
       url: https://github.com/chapulina/dolly.git
       version: foxy
     status: maintained
+  doosan-robot2:
+    doc:
+      type: git
+      url: https://github.com/doosan-robotics/doosan-robot2.git
+      version: foxy
```

I think it would need to have the complete format as this:
```suggestion
  doosan-robot2:
    doc:
      type: git
      url: https://github.com/doosan-robotics/doosan-robot2.git
      version: foxy
    source: 
      type: git
      url: https://github.com/doosan-robotics/doosan-robot2.git
      version: foxy
   status: *current_maintanaince_status*
```